backend/tests/Tests/Interpreter.Tests.fs

```diff
@@ -632,6 +632,24 @@ module Fns =
 
   let tests = testList "Fns" [ Builtin.tests; Package.tests ]
 
+module Statement =
+  let simple = t "()\n true" E.Statements.statement (RT.DInt64(1L))
+  let simple2 =
+    t "Builtin.printLine(\"hello\")\n 3L" E.Statements.statement2 (RT.DInt64(3L))
+  let nested =
+    t
+      "Builtin.printLine(\"1st statement 1st expr\")\nBuiltin.printLine(\"1st statement 2nd expr\")\nBuiltin.printLine(\"2nd statement 1st expr\")\n0L"
```

Comment:
TODO: fix this

I don't think using `Builtin.printLine` in these tests is a good idea, as it creates noise when we run the tests
example:
![image](https://github.com/user-attachments/assets/febde40b-e0e6-4b77-aea9-9da7e2aa0834)

Alternatives use another function that returns a unit --either create it, or use a package function. Hmm, maybe not, I don't think we have an easy way to access package functions from here